src/main/java/seedu/duke/model/quiz/QuizManager.java

```diff
@@ -243,8 +243,8 @@ public void delete(String[] userInputs) throws IndexOutOfBoundsException {
             return;
         }
 
-
-        userInterface.showToUser("Noted. I've removed this quiz question: \n" + quizzes.get(quizIndex - 1));
+        userInterface.showToUser("Noted. I've removed this quiz question:",
+                quizzes.get(quizIndex - 1).toString());
 
         quizzes.remove(quizIndex - 1);
```

Comment:
Maybe you can extract out 1 as a named constant. For example, naming it INDEX_OFFSET

Reply:
Added. Thanks for the suggestion.